ci-operator/config/openshift/openshift-apiserver/openshift-openshift-apiserver-release-4.5.yaml

```diff
@@ -17,9 +17,6 @@ images:
 - dockerfile_path: images/Dockerfile.rhel
   from: base
   inputs:
-    bin:
-      as:
-      - registry.svc.ci.openshift.org/ocp/builder:golang-1.12
```

Comment:
@stlaz another followup for https://github.com/openshift/openshift-apiserver/commit/b12b26e0f8ff8106bb055ae5e95370c4ddf2bbce#diff-475e714b0c3678141c59dab9961e8841R1